docs/source/residuals_diagnostics.rst

```diff
@@ -0,0 +1,30 @@
+*********************
+Residuals diagnostics
+*********************
+
+.. module:: pints.residuals_diagnostics
+
+Pints includes functionality to generate diagnostic plots of the residuals.
+These tools may be useful to evaluate the validity of a noise model.
+
+Plotting functions:
```

Comment:
Are these some kind of index?

Reply:
Yeah, here I was roughly imitating [diagnostic_plots.rst](https://github.com/pints-team/pints/blob/master/docs/source/diagnostic_plots.rst) by categorising and listing the functions at the top. It may be unnecessary given the small size of the file however.

Reply:
I could remove for now, and add later if ever the file becomes big enough that it would be helpful?

Reply:
Thanks. Now wondering if we shouldn't just merge this with the existing `diagnostics.rst`. Maybe even move the functions into the main `pints` module like we did with the other diagnostics?
Any thoughts @ben18785 ?

Reply:
https://pints.readthedocs.io/en/latest/diagnostics.html
https://pints.readthedocs.io/en/latest/diagnostic_plots.html

Reply:
Thoughts @martinjrobins ?

Reply:
probably keep them in the submodule if that is where they currently are. I don't think it matters either way